src/dev-app/dev-app/dev-app-layout.ts

```diff
@@ -117,7 +117,6 @@ export class DevAppLayout {
     {name: 'MDC Progress Bar', route: '/mdc-progress-bar'},
     {name: 'MDC Progress Spinner', route: '/mdc-progress-spinner'},
     {name: 'MDC Tabs', route: '/mdc-tabs'},
-    {name: 'MDC Tooltip', route: '/mdc-tooltip'},
```

Comment:
I think that we should keep the demos around under `/legacy-{component}` in case we need to debug anything.

Reply:
Good point - I added that back in